### ISSUE TITLE
Undertow server implementation does not support BytesContent

### ISSUE THREAD

**Issue by plokhotnyuk** [outr/youi#78]
```
scala.MatchError: BytesContent([B@78a9621a, contentType: application/json) (of class io.youi.http.content.BytesContent)
	at io.youi.server.UndertowServerImplementation$.handleStandard(UndertowServerImplementation.scala:277)
	at io.youi.server.UndertowServerImplementation$.response(UndertowServerImplementation.scala:190)
	at io.youi.server.UndertowServerImplementation.$anonfun$requestHandler$2(UndertowServerImplementation.scala:116)
	at io.youi.server.UndertowServerImplementation.$anonfun$requestHandler$2$adapted(UndertowServerImplementation.scala:115)
```
[Code to reproduce](https://github.com/plokhotnyuk/FrameworkBenchmarks/blob/add-youi-server-bytes-content/frameworks/Scala/youi/src/main/scala/example/Main.scala#L18)

**Comment by darkfrog26** [comment]
This is resolved in `0.10.2` that is being published right now.

**Comment by darkfrog26** [comment]
FYI, you can also just do:
`path"/json" / Content.json(JsonUtil.toJson(Message("Hello, World!")))`
